pylink/jlink.py

```diff
@@ -5191,3 +5191,39 @@ def cp15_register_write(self, cr_n, op_1, cr_m, op_2, value):
         if res != 0:
             raise errors.JLinkException(res)
         return res
+
+###############################################################################
+#
+# JTAG API (Raw JTAG access for boundary scan or other purposes)
+#
+###############################################################################
+
+    @interface_required(enums.JLinkInterfaces.JTAG)
+    def jtag_rawrw(self, tdo, tms, numbits=None):
```

Comment:
Could we rename this to `jtag_raw_rw` to make it a bit more clear?

Reply:
Sounds good - makes sense if adding other raw commands in the future. Updated in PR.